ndcube/ndcube.py

```diff
@@ -544,6 +561,8 @@ def axis_world_coords_values(self, *axes, pixel_corners=False, wcs=None):
         orig_wcs = wcs
         if isinstance(wcs, ExtraCoords):
             wcs = wcs.wcs
+            if not wcs:
+                return ()
```

Comment:
This was missing and is included in the other version of this method, so I added it.